src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -325,6 +334,9 @@ include: /var/unbound/etc/*.conf
 
 {$forward_conf}
 
+python:
```

Comment:
what if dnsbl is disabled?

Reply:
python is used unconditionally, the scripts itself picks up on dnsbl being enabled/disabled. a common code path makes more sense if we're looking towards the future regarding statistics.